backend/metadata/databases/default/tables/core_sections.yaml

```diff
@@ -15,45 +15,58 @@ insert_permissions:
         agreement:
           group:
             group_users:
-              user_id:
-                _eq: X-Hasura-User-Id
+              user:
+                user_id:
+                  _eq: X-Hasura-User-Id
       columns:
         - agreement_id
+        - content
```

Comment:
Sections shouldn't include content, only suggestion do. As the user creates an agreement with initial sections, an initial suggestion is created for each section. The content from the "insert new section" part is actually the content of the first suggestion of that section.

Reply:
@avirankatz This is fixed.